ark-grpc/src/client.rs

```diff
@@ -87,9 +91,13 @@ impl Client {
         let indexer_client = IndexerServiceClient::connect(self.url.clone())
             .await
             .map_err(Error::connect)?;
+        let admin_client = AdminServiceClient::connect(self.url.clone())
+            .await
+            .map_err(Error::connect)?;
```

Comment:
Hmmm, I think we should expose a separate admin client. An `ark_grpc::Client` in production will never use an admin client.

I would just create a separate module `admin_client.rs` and move all the relevant bits there.

Reply:
nice idea thanks!